src/store/view.js

```diff
@@ -253,8 +253,14 @@ export default function viewState(state = null, action) {
     }
     else if (action.type == types.MERGE_PERSON_DUPLICATES + '_FULFILLED') {
         // Close the relevant pane
+        let openPanes = state.panes.filter(paneData => paneData.id != action.meta.paneId);
```

Comment:
Instead of first closing and then opening, it's better to `map()` all panes, ignoring those that are irrelevant to the operation and replacing the closing one with the new one. Why? Because that way, it will always be opened in the right place. The way it's implemented now in this patch, the new pane will always be the last one, i.e. opened furthest to the right, even if the merge pane is not the rightmost one.